docs/api/fallback/routes.js

```diff
@@ -3,19 +3,32 @@
  * Licensed under the MIT License.
  */
 
-// Map of incoming URL paths to redirect URLs
 const routes = [
 	// We previously served the API docs out of `docs/apis`.
-	// Forward to "current" version of the API docs (`docs/api/v2`).
-	{ from: "/docs/apis", to: "/docs/api/v2" },
+	// Forward to current version of the API docs (`docs/api`).
+	{ from: "/docs/apis", to: "/docs/api" },
```

Comment:
Redirects don't seem to be working on https://lively-bay-0171da81e-preview.westus2.5.azurestaticapps.net/, is that expected?

Reply:
That is expected, yes. They work correctly locally via `serve-with-azure-emulation` script, and I've confirmed that the pattern used here works correctly on the current production site.